MusicApp.SongService.Web/Controllers/SongsController.cs

```diff
@@ -0,0 +1,62 @@
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using MusicApp.SongService.Application.CQRS.Commands.CreateSong;
+using MusicApp.SongService.Application.CQRS.Commands.DeleteSong;
+using MusicApp.SongService.Application.CQRS.Commands.EnsureArtistCreated;
+using MusicApp.SongService.Application.CQRS.Commands.UpdateSong;
+using MusicApp.SongService.Application.CQRS.Queries.GetSongById;
+using MusicApp.SongService.Application.CQRS.Queries.GetSongs;
+using MusicApp.SongService.Application.DTOs;
+
+namespace MusicApp.SongService.Web.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class SongsController : ControllerBase
+{
+    private readonly IMediator _mediator;
+
+    public SongsController(IMediator mediator)
+    {
+        _mediator = mediator;
+    }
+    
+    [HttpGet]
+    public async Task<IActionResult> GetSongs(CancellationToken cancellationToken)
+    {
+        return Ok(await _mediator.Send(new GetSongsQuery(), cancellationToken));
```

Comment:
Again, it's better to separate it. Create GetSongsQuery(), then call mediator, then return result